mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -37,6 +37,8 @@
 #include "llvm/Support/ToolOutputFile.h"
```

Comment:
Could you update the `conv2d_harness_cpu_verifier.mlir` according to the change you have here? It can be a good reference for end results you have achieved.

Reply:
Will do